docs/docs/50-user-guide/60-reference-docs/30-promotion-steps/send-message.md

```diff
@@ -83,6 +84,23 @@ steps:
       }) }}
 ```
 
+## Outputs
+
+The `send-message` step returns different outputs depending on the type of channel used. Each output
+is nested under a key corresponding to the channel kind (e.g., `slack`, `smtp`).
+
+### Slack Output
+
+When sending a message to a Slack channel, the output will include the following fields:
+
+| Name       | Type   | Description                                                                                                                       |
+| ---------- | ------ | --------------------------------------------------------------------------------------------------------------------------------- |
+| `threadTS` | string | The timestamp of the message that started the thread, if applicable. Otherwise returns the current threadTS if it was already set |
```

Comment:
I wonder if something like this might be more clear:

> [..] When starting a new thread, the timestamp of the newly created message. When replying to an existing thread, the provided threadTS.